src/streamlink/plugins/nrk.py

```diff
@@ -13,7 +13,7 @@
 
 _id_re = re.compile(r"/(?:program|direkte|serie/[^/]+)/([^/]+)")
 _url_re = re.compile(r"https?://(tv|radio).nrk.no/")
-_api_baseurl_re = re.compile(r'apiBaseUrl:\s*"(?P<baseurl>[^"]+)"')
+_api_baseurl_re = re.compile(r'''apiBaseUrl:\s*["'](?P<baseurl>[^"']+)["']''')
```

Comment:
i think this is an overkill with **\1**, it's maybe useful for longer regex but this one is already short.